server/container_list.go

```diff
@@ -84,7 +84,7 @@ func (s *Server) ListContainers(ctx context.Context, req *types.ListContainersRe
 		switch cState.Status {
 		case oci.ContainerStateCreated:
 			rState = types.ContainerState_CONTAINER_CREATED
-		case oci.ContainerStateRunning:
+		case oci.ContainerStateRunning, oci.ContainerStatePaused:
```

Comment:
herer I check the code of containerd, u can check https://github.com/containerd/containerd/blob/77d53d2d230c3bcd3f02e6f493019a72905c875b/pkg/cri/store/container/status.go#L103 @haircommander

Reply:
intersting, I could be compelled by this. @saschagrunert WDYT

Reply:
if its unknow state, kubelet will delete the paused ctr and restart one new.

Reply:
I think hiding the information from the kubelet is the only way to make it work. It would require a kubernetes change to display the paused statuts to the end-user.

Reply:
I think this is the reason why containrd do like this

Reply:
This just seems wrong, but I'll let you all noodle it out.